src/main/java/DukeException.java

```diff
@@ -0,0 +1,49 @@
+public class DukeException extends Exception {
+
+    public String getMessage() {
+        return toString();
+    }
+}
+
+class EmptyCommandException extends DukeException {
+    private String type;
+
+    EmptyCommandException(String type) {
+        this.type = type;
+    }
+
+    @Override
+    public String toString() {
+        if (this.type.equals("event")) {
```

Comment:
Maybe it's better to check if type starts with a vowel instead of hardcoding the "event" case